opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -633,6 +633,59 @@ const std::vector<int>& CpGrid::globalCell() const
     return currentData().back() -> global_cell_;
 }
 
+void CpGrid::computeGlobalCellLgr(const int& level, const std::array<int,3>& startIJK, std::vector<int>& global_cell_lgr)
+{
+    assert(level);
+    for (const auto& element : elements(levelGridView(level))) {
+        // Element belogns to an LGR, therefore has a father. Get IJK of the father in the level grid the father was born.
+        // For CARFIN, parent cells belong to level 0.
+        std::array<int,3> parentIJK = {0,0,0};
+        currentData()[element.father().level()]->getIJK(element.father().index(), parentIJK);
+        // Each parent cell has been refined in cells_per_dim[0]*cells_per_dim[1]*cells_per_dim[2] child cells.
+        // element has certain 'position' inside its parent cell that can be described with 'IJK' indices, let's denote them by ijk,
+        // where 0<= i < cells_per_dim[0], 0<= j < cells_per_dim[1], 0<= k < cells_per_dim[2].
+        const auto& cells_per_dim = currentData()[level]->cells_per_dim_;
```

Comment:
I see that it might seem that we do not need it. Maybe there is a way to get rid of it as a data member. I'll think about it. 
In any case, we do use it, for example, in Entity::geometryInFather()[ here](https://github.com/OPM/opm-grid/blob/36069f06e5fe321c2dd161a7614dc95be5d05978/opm/grid/cpgrid/Entity.hpp#L535)

If we keep it, we can also change the name to cells_per_dim_per_parent_cell_. What do you think?